src/13_file_io.py

```diff
@@ -10,9 +10,18 @@
 
```

Comment:
Good job with this one. But I think if you write functions for this, that would make it more readable.